gwpy/table/io/gravityspy.py

```diff
@@ -92,10 +92,14 @@ def get_gravityspy_triggers(tablename, engine=None, **kwargs):
 
 def get_connection_str(db='gravityspy',
                        host='gravityspy.ciera.northwestern.edu',
-                       user=os.getenv('GRAVITYSPY_DATABASE_USER', None),
-                       passwd=os.getenv('GRAVITYSPY_DATABASE_PASSWD', None)):
+                       user=None,
+                       passwd=None):
     """Create string to pass to create_engine
     """
+    if (not user) or (not passwd):
+        user = os.getenv('GRAVITYSPY_DATABASE_USER', None)
+        passwd = os.getenv('GRAVITYSPY_DATABASE_PASSWD', None)
```

Comment:
@scottcoughlin2014, is it intention that `user` and `passwd` are _both_ required, otherwise are ignored? If so, can this be made clear in the docstring? Otherwise we should parse each separately.

Reply:
Yes both are needed, I will mkae clear